src/app-layout/utils/use-drawers.ts

```diff
@@ -50,12 +52,22 @@ function useRuntimeDrawers(
 ) {
   const [runtimeDrawers, setRuntimeDrawers] = useState<DrawersLayout>({ before: [], after: [] });
 
+  const prevDrawersRef = useRef<Array<RuntimeDrawerConfig>>([]);
+
   useEffect(() => {
     if (disableRuntimeDrawers) {
       return;
     }
     const unsubscribe = awsuiPluginsInternal.appLayout.onDrawersRegistered(drawers => {
       setRuntimeDrawers(convertRuntimeDrawers(drawers));
+      if (prevDrawersRef.current) {
+        const diff = drawers.filter(drawer => !prevDrawersRef.current.find(d => drawer.id === d.id));
```

Comment:
With it, the newly registered drawer will open even its priority is lower than the current open one. Is it expected?